packages/ui/src/components/va-time-input/VaTimeInput.vue

```diff
@@ -115,7 +115,7 @@ import VaIcon from '../va-icon/VaIcon.vue'
 import { VaDropdown, VaDropdownContent } from '../va-dropdown'
 
 const VaInputProps = extractComponentProps(VaInput, [
-  'mask', 'returnRaw', 'autosize', 'minRows', 'maxRows', 'type', 'inputmode',
+  'mask', 'returnRaw', 'autosize', 'minRows', 'maxRows', 'type', 'inputmode', 'counter', 'maxLength',
```

Comment:
We will need to move VaTimeInput and VaDateInput from input, to VaInputWrapper in other task.